shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.h

```diff
@@ -62,6 +63,11 @@ FLUTTER_DARWIN_EXPORT
 @property(nonatomic, copy) NSDictionary* markedTextStyle;
 @property(nonatomic, assign) id<UITextInputDelegate> inputDelegate;
 
+// Delta
+@property(nonatomic, strong) FlutterTextEditingDelta* currentTextEditingDelta;
```

Comment:
nit: `strong` is implied. Maybe this should be `readonly`?

Reply:
IIRC I wasn't able to use setters when it was `readonly`.